yarn-project/sequencer-client/src/global_variable_builder/global_builder.ts

```diff
@@ -46,6 +46,10 @@ export class GlobalVariableBuilder implements GlobalVariableBuilderInterface {
     });
   }
 
+  public async getCurrentBaseFees(): Promise<GasFees> {
```

Comment:
Is there an issue to set the DA fee?

Reply:
There is not, it is just an initial simple model. From the description of the model: https://github.com/AztecProtocol/engineering-designs/blob/main/in-progress/8757-fees/design.md
![image](https://github.com/user-attachments/assets/ce2d9c32-bd47-4430-b762-6348a8cc74ad)